helm-go-lib/main.go

```diff
@@ -158,10 +164,16 @@ func installChart(namespace string, kubeToken string, apiServer string, releaseN
 }
 
 //export uninstallRelease
-func uninstallRelease(namespace, kubeToken, apiServer, releaseName string) *C.char {
-	settings := cli.New()
-	actionConfig := new(action.Configuration)
-	err := actionConfig.Init(settings.RESTClientGetter(), namespace, os.Getenv("HELM_DRIVER"), glog.Infof)
+func uninstallRelease(namespace string, kubeToken string, apiServer string, caFile string, releaseName string) *C.char {
+	var kubeConfig *genericclioptions.ConfigFlags
+	kubeConfig = genericclioptions.NewConfigFlags(false)
+	kubeConfig.APIServer = &apiServer
+	kubeConfig.BearerToken = &kubeToken
+	kubeConfig.CAFile = &caFile
+	kubeConfig.Namespace = &namespace
+
+    actionConfig := new(action.Configuration)
+	err := actionConfig.Init(kubeConfig, namespace, os.Getenv("HELM_DRIVER"), glog.Infof)
 	if err != nil {
 		return C.CString(err.Error())
 	}
```

Comment:
I wonder if we should factor out this code block as it's common in all API methods 🤔